keyboards/kbdcraft/adam0110/info.json

```diff
@@ -0,0 +1,245 @@
+{
+    "manufacturer": "KBDCRAFT",
+    "keyboard_name": "Adam0110",
+    "maintainer": "KBDCRAFT",
+    "bootloader": "tinyuf2",
+    "diode_direction": "ROW2COL",
+    "eeprom": {
+        "wear_leveling": {
+            "backing_size": 16384
+        }
+    },
```

Comment:
[Drashna suggested that this value be used as a safe measure](https://discord.com/channels/440868230475677696/440868230475677698/1185448835758162031) since the exact variant of the F402 these boards actually used (configured as F401) is currently unknown.

Reply:
Think I forgot to mention that without this, the `via` keymap will fail to compile.

Reply:
Ah I misinterpreted the default value, soz.